spec/mailers/jobseekers/alert_mailer_spec.rb

```diff
@@ -61,28 +61,28 @@
     let(:frequency) { "daily" }
 
     it "sends a job alert email" do
-      expect(mail.subject).to eq(I18n.t("alert_mailer.alert.subject"))
+      expect(mail.subject).to eq(I18n.t("jobseekers.alert_mailer.alert.subject"))
       expect(mail.to).to eq([subscription.email])
-      expect(body).to include(I18n.t("alert_mailer.alert.summary.daily", count: 1))
```

Comment:
I would love if you could replace most of the `expect(body).to`s in this scenario with `and`.

Reply:
Probably just punishment for a PR with so many file changes 😂